package.json

```diff
@@ -118,7 +118,6 @@
     "fast-levenshtein": "^2.0.6",
     "file-loader": "^1.1.11",
     "fuse.js": "^3.2.0",
-    "gulp": "github:gulpjs/gulp#4.0",
```

Comment:
Is this a safe change? Does a fresh install still let you build, if you don't have `gulp` globally installed?